elasticdl/python/worker/worker.py

```diff
@@ -509,6 +511,7 @@ def _run_model_call_before_training(self, features):
               more than once during one forward-pass.
         """
         if self._embedding_layers:
+            self.report_embedding_info()
```

Comment:
Would it be better to put this line into function `_init_embedding_layer `? There are two reasons:
* `_run model_call_before_training` does some initialization that needs to do after `model.call` is called. However, reporting embedding info is a kind of initialization that is not dependent on `model.call`.
* If a user calls `worker.set_model()` to change a model in unit tests, he doesn't need to call `report_embedding_info` manually.

Reply:
Done